server/routes/api/browser.py

```diff
@@ -64,42 +64,90 @@ def get_property_labels(dcid):
     return Response(json.dumps(labels), 200, mimetype='application/json')
 
 
-@bp.route('/observation-ids')
-@cache.cached(timeout=3600 * 24, query_string=True)  # Cache for one day.
-def get_observation_ids():
-    """Returns a mapping of date to observation node dcid for a combination of predicates:
-    observedNodeLocation, statisticalVariable, measurementMethod (optional), observationPeriod (optional)"""
-    place_id = request.args.get("place")
-    if not place_id:
-        return Response(json.dumps("error: must provide a place field"),
-                        400,
-                        mimetype='application/json')
-    stat_var_id = request.args.get("statVar")
-    if not stat_var_id:
-        return Response(json.dumps("error: must provide a statVar field"),
-                        400,
-                        mimetype='application/json')
-    measurement_method = request.args.get("measurementMethod", "")
-    observation_period = request.args.get("obsPeriod", "")
+def get_sparql_query(place_id, stat_var_id, date, measurement_method,
+                     observation_period):
     measurement_method_triple = ""
     if measurement_method:
         measurement_method_triple = f"""?svObservation measurementMethod {measurement_method} ."""
```

Comment:
nit: you don't need a multiline string for this (here and below), f"..." should be sufficient

Reply:
done